dev/sparktestsupport/modules.py

```diff
@@ -256,9 +256,21 @@ def __hash__(self):
 )
 
 
+mllib_local = Module(
+    name="mllib-local",
+    dependencies=[sketch],
```

Comment:
It doesn't belong in either sketch or mllib-local really. Since the purpose is to dissociate from spark-core, you should probably make a new exception type or examine whether generic exception types can be used instead.